.github/workflows/docs-links.yaml

```diff
@@ -0,0 +1,22 @@
+name: docs links on maas.io/docs
+
+on: pull_request
+
+jobs:
+  check-links:
+    if: github.repository == 'canonical-web-and-design/maas.io'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Install linkchecker
+        run: sudo pip install LinkChecker
+
+      - name: Run linkchecker
+        run: linkchecker --threads 2 --ignore-url "https://assets.ubuntu.com$" --check-extern --no-warnings https://maas.io/docs
```

Comment:
why only 2 threads? Default appears to be 10

Reply:
I took that from other ones we have. The default is 100 :)

Reply:
https://linkchecker.github.io/linkchecker/man/linkchecker.html#cmdoption-t ?

Reply:
That seems like an old man page you have there.

https://github.com/linkchecker/linkchecker/blob/365852a4428bf9fe7e58fa0f7f2845719d531182/doc/changelog.txt#L212